staging/src/k8s.io/client-go/util/certificate/certificate_manager.go

```diff
@@ -235,14 +236,20 @@ func NewManager(config *Config) (Manager, error) {
 		getTemplate = func() *x509.CertificateRequest { return config.Template }
 	}
 
```

Comment:
as a safety check, if config.GetUsages and config.Usages are both set, return a usage error here (they are mutually exclusive)